include/glow/Optimizer/GraphOptimizer/GraphOptimizer.h

```diff
@@ -29,12 +30,30 @@ class Module;
 class PlaceholderBindings;
 class Placeholder;
 
+/// \returns the default graph optimization pipeline.
+FunctionPassPipeline getDefaultGraphOptimizationPipeline();
```

Comment:
If I understand it correctly, this "DefaultGraphOptimizationPipeline" is not target independent, and it contains all passes. But it is not clear to me in which case it should be called. "getTargetIndependentGraphOptimizationPipeline" should be the common opt pass. For users, I think they should add something in IndependentGraphOptimization pass?

Reply:
Yeah I wasn't sure the best design here. I was thinking all backends would start with the pipeline with all passes ("DefaultGraphOptimizationPipeline") and then opt out of passes they didn't want via `OptimizationOptions ::funPassesToSkip`. But it could work in reverse, by adding just the TargetIndependent ones and then adding which additional passes they want.

Reply:
IMO, adding passes may be more easier and debuggable.  Once a pass is added and tested, it is kind of reliable. Then another pass can be added, and so on.